index.html

```diff
@@ -17,7 +17,7 @@
 
     <title>Shoppy</title>
   </head>
-  <body>
+  <body data-theme="stitch">
```

Comment:
Helps to apply the default styles before the JS loads, as the first theme is Stitch.

Reply:
Our CSS detects with `body[data-theme="stitch"]` and uses `document.body.setAttribute("data-theme", site)` to update, so I think I'll leave the same property as the JavaScript wouldn't have been loaded yet, and we wouldn't be able to duplicate the CSS selectors 🙏🏻